sched/group/group_setuptaskfiles.c

```diff
@@ -57,16 +57,14 @@
 int group_setuptaskfiles(FAR struct task_tcb_s *tcb)
 {
   FAR struct task_group_s *group = tcb->cmn.group;
+  uint8_t ttype = tcb->cmn.flags & TCB_FLAG_TTYPE_MASK;
 #ifndef CONFIG_FDCLONE_DISABLE
   FAR struct tcb_s *rtcb = this_task();
   int ret;
 #endif
 
   DEBUGASSERT(group);
-#ifndef CONFIG_DISABLE_PTHREAD
-  DEBUGASSERT((tcb->cmn.flags & TCB_FLAG_TTYPE_MASK) !=
-              TCB_FLAG_TTYPE_PTHREAD);
-#endif
+  DEBUGASSERT(ttype != TCB_FLAG_TTYPE_PTHREAD);
```

Comment:
Why `#ifndef CONFIG_DISABLE_PTHREAD` is removed?

Reply:
@pkarashchenko 
I think the original code is incorrect. The condition should be checked always.